src/tike/operators/ptycho.py

```diff
@@ -33,14 +30,27 @@ class Ptycho(Operator):
         The pixel width and height of the reconstructed grid.
     ntheta : int
         The number of angular partitions of the data.
+    model : string
+        The type of noise model to use for the cost functions.
+    propagation : Operator
+        The wave propagation operator being used.
+    diffraction : Operator
+        The object probe interaction operator being used.
 
     Parameters
     ----------
     psi : (ntheta, nz, n) complex64
         The complex wavefront modulation of the object.
```

Comment:
is this OxP or simply O?

Reply:
It's just O. It should probably be renamed obj. `psi` is a holdover from notation in a paper years ago.